templates/core/terraform/routetable/routetable.tf

```diff
@@ -0,0 +1,18 @@
+resource "azurerm_route_table" "rt" {
+  name                          = "rt-${var.resource_name_prefix}-${var.environment}-${var.tre_id}"
+  resource_group_name           = var.resource_group_name
+  location                      = var.location
+  disable_bgp_route_propagation = false
+
+  route {
+    name           = "DefaultRoute"
```

Comment:
Do the packets intended for the subnet also take a hop from the firewall?

Reply:
AFAIK no, @marrobi if you want to elaborate?

Reply:
Traffic on the same subnet will not need to be routed so no.

Also route tables are evaluated most specific to least specific - so this is  only for traffic without a more specific route defined - https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-udr-overview#how-azure-selects-a-route

So get something like: 

![image](https://user-images.githubusercontent.com/17089773/117622398-83c04e00-b16a-11eb-9bad-6c920b4694ae.png)